civiform/civiform
# Publish all programs modal - update formatting and add information about universal questions

### Description

This PR updates formatting for the "Publish all programs" modal to better match [these designs](https://www.figma.com/file/KqfHZYPS3Ffd93aJsfkNYZ/Universal-Questions?type=design&node-id=1-19&mode=design&t=CVlR9wsQFdFk8ADa-0). It also adds information about the number of universal questions contained within each program (hidden behind a feature flag).

## Release notes

### Checklist

#### General



#### New Features


### Instructions for manual testing

1) Login as an admin
2) Enable the "UNIVERSAL_QUESTIONS" feature flag
2) Create a draft program
3) Create a draft question and mark it as universal
4) Click the "Publish all drafts" button on the program list page
5) See the modal with message about universal questions ("Contains 0 of 1 universal questions")
6) Close the modal and add the question to the program
7) Click the "Publish all drafts" button on the program list page
8) See the modal with the message "Contains all universal questions"

### Issue(s) this completes

Fixes #6096
